### PR TITLE
Sms

### PR DESCRIPTION
**Pull request by LiamMillerGofore**
- The SMS service has been tested using a real phonenumber with Mikeal.
- The service will not work in dev mode to prevent misuse.
- Additional features have been made to only work in dev mode.
- The SMS service itself does not have a dedicated test file as the service is set to only run on prod.
- A number of environment variables will be provided to the live environment to let this run.
- Once feedback is given with regards to the sender name, we shall change this, but do not want to block the task over it currently.

### CODE REVIEW

**Comment by LiamMillerGofore** [comment]
I have considered an issue since, please do not review until a discussion has taken place regarding localised vs international phone numbers.

**Comment by LiamMillerGofore** [comment]
The issue regarding phone numbers should now be resolved, all phone numbers are transformed to 358 on DB entry.

**Comment by LiamMillerGofore** [comment]
To test the SMS service itself, contact me and I can provide access to the service.

File: backend/src/sms/sms.service.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            });
+
+    }
+
+    private getOneTimeLink(challenge: Challenge): string {
+        return `${ConfigHelper.getFrontendPort()}/login?challenge=${challenge.challenge}&id=${challenge.id}`;
+    }
+
+    private getMessage(recipientName: string, systemName: string, link: string) {
+        return `Hei ${recipientName}! Sinulle on luotu oma ${systemName}-tili. Voit kirjautua palveluun kertakäyttöisen kirjautumislinkin avulla ${link}.`;
```
Comment (Maindii) [inline backend/src/sms/sms.service.ts]: Should this be in content.json?

**Pull request merged**